include/Win.h

```diff
@@ -28,29 +29,38 @@ struct GmWin : public CoreNode {
 
 	GmWin(); // unused/inlined
 
-	void open();
-	void close();
-	void render(Graphics&);
-	void placeCentre();
-	void print(Graphics&, int, int, char*);
-	void printcentre(Graphics&, int, char*);
-	void printright(Graphics&, int, char*);
-	void printleft(Graphics&, int, char*);
-	void printStart(Graphics&);
-	void texture(Graphics&, Texture*, int, int, int, int, RectArea);
-	void texturecentre(Graphics&, Texture*, int, int, int, RectArea);
-	void textureleft(Graphics&, Texture*, int, int, int, RectArea);
-	void textureright(Graphics&, Texture*, int, int, int, RectArea);
-
-	// unused/inlined:
-	void setRect(int, int);
-	void moveHome(Vector2i&);
-	void doRender(Graphics&);
-	void update();
+	virtual void open();                                                      // _10
+	virtual void close();                                                     // _14
+	virtual void update();                                                    // _18 unused/inlined
+	virtual void doRender(Graphics&);                                         // _1C unused/inlined
+	virtual void render(Graphics&);                                           // _20
+	virtual void printStart(Graphics&);                                       // _24
+	virtual void print(Graphics&, int, int, char*);                           // _28
+	virtual void printcentre(Graphics&, int, char*);                          // _2C
+	virtual void printleft(Graphics&, int, char*);                            // _30
+	virtual void printright(Graphics&, int, char*);                           // _34
+	virtual void texture(Graphics&, Texture*, int, int, int, int, RectArea);  // _38
+	virtual void texturecentre(Graphics&, Texture*, int, int, int, RectArea); // _3C
+	virtual void textureleft(Graphics&, Texture*, int, int, int, RectArea);   // _40
+	virtual void textureright(Graphics&, Texture*, int, int, int, RectArea);  // _44
 
-	// _00     = VTBL?
-	// _00-_14 = CoreNode?
-	// TODO: members
+	void placeCentre();
+	void setRect(int, int);   // unused/inlined
+	void moveHome(Vector2i&); // unused/inlined
+
+	CloseListener* mCloseListener; // _14
+	int mWidth;                    // _18
+	int mHeight;                   // _1c
+	int mPosX;                     // _20
+	int mPosY;                     // _24
+	int mStatus;                   // _28
+	int m2c;                       // _2c
```

Comment:
also, for the unknown/unnamed member variables, we usually keep a convention of naming them `int _2C;` until we're sure, so if you wouldn't mind updating these in line with that, that'd be perfect. the rest of the names seem good though!